README.md

```diff
@@ -1,74 +1,76 @@
-[![Stargazers repo roster for @jdeath/Hubspace-Homeassistant](https://git-lister.onrender.com/api/stars/jdeath/Hubspace-Homeassistant?limit=30)](https://github.com/jdeath/Hubspace-Homeassistant/stargazers)
+# Overview
+This repository provides an integration between Hubspace and Home Assistant. Due to the cloud-first
+nature of Hubspace devices, an internet connection needs to be available, along with the Hubspace servers.
 
 Please ask support questions in homeassistant forums: https://community.home-assistant.io/t/add-support-for-hubspace-by-afero/306645
```

Comment:
@jdeath Is this still the preferred way of asking questions, or is it GitHub issues?